pipeline-examples/parallel-multiple-nodes/ParallelMultipleNodes.groovy

```diff
@@ -0,0 +1,12 @@
+def labels = ['precise', 'trusty'] // labels for Jenkins node types we will build on
+def builders = [:]
+for (x in labels) {
```

Comment:
So it's a Groovy scoping issue inside the nested closure? Jesse's comment doesn't actually explain anything to me :frowning: 

Anyways, I suggest offering a little comment explaining why the variable needs to be bound, lest somebody like me remove it and be confused when things explode